Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -82,6 +82,9 @@
 static NSString * const lastSeenVersionStateKey = @"LastSeenVersion";
 static NSString * const lastLoadedDNRHashStateKey = @"LastLoadedDNRHash";
 
+// FIXME: Share this between here and WebExtensionContextAPIDeclarativeNetRequestCocoa.
+static NSString * const declarativeNetRequestModifiedRulesetStateKey = @"DeclarativeNetRequestRulesetState";
```

Comment:
Instead of needing to share it, maybe it's better to put:
`[m_state removeObjectForKey:declarativeNetRequestModifiedRulesetStateKey];` into a helper method that is implemented in `WebExtensionContextDeclarativeNetRequsetCocoa.mm`?